doc/user/content/self-hosted/_index.md

```diff
@@ -0,0 +1,242 @@
+
+### `kubectl`
+
+This tutorial uses `kubectl`. To install, refer to the [`kubectl` documentationq](https://kubernetes.io/docs/tasks/tools/).
+
+### Kubernetes Storage Configuration
+
+Materialize requires fast, locally-attached NVMe storage for optimal performance. Network-attached storage (like EBS volumes) can significantly degrade performance and is not supported.
+
+We recommend using OpenEBS with LVM Local PV for managing local volumes. While other storage solutions may work, we have tested and recommend OpenEBS for optimal performance.
+
+#### Install OpenEBS
+
+Install OpenEBS to your running Kubernetes cluster.
+
+```bash
+# Add OpenEBS to Helm
+helm repo add openebs https://openebs.github.io/openebs
+helm repo update
+
+# Install only the Local PV Storage Engines
+helm install openebs --namespace openebs openebs/openebs \
+  --set engines.replicated.mayastor.enabled=false \
+  --create-namespace
+```
+
+Verify the installation:
+```bash
+kubectl get pods -n openebs -l role=openebs-lvm
+```
+
+#### LVM Configuration
+
+LVM setup varies by environment. Below is our tested and recommended configuration:
+
+##### AWS EC2 with Bottlerocket AMI
+Tested configurations:
+- Instance types: r6g, r7g families
+- AMI: AWS Bottlerocket
+- Instance store volumes required
+
+Setup process:
+1. Use Bottlerocket bootstrap container for LVM configuration
+2. Configure volume group name as `instance-store-vg`
+
+**Note:** While LVM setup may work on other instance types with local storage (like i3.xlarge, i4i.xlarge, r5d.xlarge), we have not extensively tested these configurations.
+
+#### Storage Configuration
+
+Once LVM is configured, set up the storage class:
+
+{{< tabs >}}
+{{< tab "OpenEBS" >}}
+
+If using OpenEBS, set up the storage class as follows:
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "openebs-lvm-instance-store-ext4"
+    provisioner: "local.csi.openebs.io"
+    parameters:
+      storage: "lvm"
+      fsType: "ext4"
+      volgroup: "instance-store-vg"
+```
+{{< /tab >}}
+{{< tab "Other Storage" >}}
+While OpenEBS is our recommended solution, you can use any storage provisioner that meets your performance requirements by overriding the provisioner and parameters values.
+
+For example, to use a different storage provider:
+
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "your-storage-class"
+    provisioner: "your.storage.provisioner"
+    parameters:
+      # Parameters specific to your chosen storage provisioner
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Install the chart
+
+1. If installing for the first time, create a namespace. The default
+   configuration uses the `materialize` namespace.
+
+   ```bash
+   kubectl create namespace materialize
+   ```
+
+1. Set up the Materialize operator Helm repository.
+
+   a. <red>TBD whether this is needed</red>. Add Helm to install charts that are
+      hosted in the Materialize operator Helm repository:
```

Comment:
I wasn't sure if we'd be having people add materialize or we expect people to just have the repo so that they can just install.

Reply:
right now the helm chart is just in the materialize repo... at some point soon we'll host it on a separate endpoint. for now I think we just tell people to download the materialize repo, check out a specific tag, and add the local path